README.md

```diff
@@ -30,6 +49,10 @@ $ git machete traverse
   # (put each branch one by one in sync with its parent and remote counterpart)
 ```
 
+## Contribute
+
+Use `tox -e venv` to setup virtual environment to work on that project in your favorite IDE. Use `.tox/venv/bin/python` as a reference `python` interpreter in your IDE.
```

Comment:
```
pip install --user tox
tox
```
right?

Reply:
Ok, I'm getting https://stackoverflow.com/questions/21530577/fatal-error-python-h-no-such-file-or-directory... maybe it's worth mentioning that python header files need to be installed, as in the most upvoted answer there

Reply:
Whoops... since setup.py has been run under sudo, now I've got a bunch of files belonging to root:

```
total 180
drwxrwxr-x 10 plipski plipski  4096 lut  5 20:38 ./
drwxr-xr-x 91 plipski plipski  4096 lut  5 20:19 ../
-rw-r--r--  1 root    root      147 lut  5 20:27 AUTHORS
drwxr-xr-x  3 root    root     4096 lut  5 20:27 build/
-rw-r--r--  1 root    root     4924 lut  5 20:27 ChangeLog
drwxr-xr-x  3 root    root     4096 lut  5 20:27 .eggs/
drwxr-xr-x  2 plipski plipski  4096 lut  5 20:26 extras/
drwxrwxr-x  8 plipski plipski  4096 lut  5 20:50 .git/
-rw-r--r--  1 plipski plipski   116 lut  3 00:14 .gitignore
drwxr-xr-x  3 plipski plipski  4096 lut  5 20:26 git_machete/
-rwxr-xr-x  1 plipski plipski 72978 lut  4 19:42 git-machete*
drwxr-xr-x  2 root    root     4096 lut  5 20:27 git_machete.egg-info/
``` 

...which means that I need to `chown` them before non-sudoed `tox` can successfully run :/ Yeah, it'd be nice if the entire setup.py could run without root privileges :)

Reply:
Yeap, `tox` is not default and needs to be installed. `pip` will do it.

Installation of `tox` depends on how python env is installed. I am recommending to use [pyenv](https://github.com/pyenv/pyenv) in such cases. It will not require `sudo` and it has python headers included by default. If python is installed from packages, then `sudo apt install python-dev` is needed.

Regarding testing, I will add paragraph about it. That's worth to mention.

Reply:
Ok thx